File: 734A.py
#139675211	Dec/18/2021 00:28UTC+5.5	hahaharshilpp	A - Anton and Danik	Python 3	Accepted	62 ms	100 KB
#https://codeforces.com/contest/734/problem/A

n = int(input())
main = input()
i = 0

AntonScore = 0
DanikScore = 0

for i in range(n):
    if main[i] == "A":
        AntonScore += 1
    else:
        DanikScore += 1

if AntonScore < DanikScore:
    print("Danik")
elif AntonScore == DanikScore:
    print("Friendship")
else:
    print("Anton")

File: 734A.cpp
//139677142	Dec/18/2021 00:53UTC+5.5	hahaharshilpp	A - Anton and Danik	GNU C++20 (64)	Accepted	15 ms	200 KB
//https://codeforces.com/contest/734/problem/A



#include <bits/stdc++.h>

using namespace std;

int main(){
    long long n; cin >> n;
    string results; cin >> results;
    
    int A = 0,D = 0;
    for(int i = 0; i < n; i++){
        if(results[i] == 'A'){
            A++;
        }else{
            D++;
        }
    }

    if(A < D){
        cout << "Danik" << endl;
    }else if(A == D){
        cout << "Friendship" << endl;
    }else{
        cout << "Anton" << endl;
    }

}
